appIndicator.js

```diff
@@ -679,7 +679,7 @@ class AppIndicators_IconActor extends St.Icon {
     _setOpacity() {
         let opacityValue = this._settings.get_int('icon-opacity');
 
-        this.opacity = opacityValue;
+        this.set_opacity(opacityValue);
```

Comment:
Well, there's no much difference honestly... In gnome-shell upstream code we tend to use properties instead.

It's true that properties don't give errors if you try to call something non-existent.

So fine as this